front/app/api/internal_comments/useInternalComments.ts

```diff
@@ -56,11 +57,6 @@ const useInternalComments = (
 
       return hasNextPage && pageNumber ? pageNumber + 1 : null;
     },
-    enabled:
-      !!parameters.userId ||
-      !!parameters.initiativeId ||
-      !!parameters.ideaId ||
-      !!parameters.commentId,
```

Comment:
We always have one of these (depending on the type, so not reason to keep these).